feature-libs/checkout/b2b/components/checkout-payment-type/checkout-payment-type.component.spec.ts

```diff
@@ -109,52 +97,49 @@ describe('CheckoutPaymentTypeComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should get all supported payment types', () => {
-    component.paymentTypes$
-      .subscribe((types) => {
-        expect(types).toBe(mockPaymentTypes);
-      })
-      .unsubscribe();
+  it('should get the loading state when selecting a payment type', (done) => {
+    component.selectedPaymentTypeIsLoading$
```

Comment:
maybe we can add a test where we would check how the component html behaves when the state is loading? i.e. check for the spinner